kuksa_apps/kanto/config/edge_client.ini

```diff
@@ -0,0 +1,11 @@
+[SERVER]
```

Comment:
If that's the intended Kuksa VAL server configuration, please, keep the .ini sections naming aligned with the already adopted pattern in all the other kuksa_apps implementations - i.e. `kuksa_val` with an `ip` property in it.

For reference - see e.g. [the Dapr app cofig.ini](https://github.com/boschglobal/kuksa.val/blob/kanto-kuksa-bridge/kuksa_apps/dapr/pubsub/config.ini).